decidim-assemblies/app/views/decidim/assemblies/admin/assemblies/index.html.erb

```diff
@@ -1,7 +1,9 @@
 <div class="card" id="assemblies">
   <div class="card-divider">
     <h2 class="card-title">
-      <%= t "decidim.admin.titles.assemblies" %><%= link_to t("actions.new", scope: "decidim.admin", name: t("models.assembly.name", scope: "decidim.admin")), ["new", "assembly"], class: "button tiny button--title" if can? :create, Decidim::Assembly %>
+      <%= link_to "#{translated_attribute(parent_assembly.title)} > ", edit_assembly_path(parent_assembly) if parent_assembly %>
```

Comment:
Are we only having 1 level of parent-child? If we're having multiple levels of parents, then this should reflect all the ancestors, I guess?